ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/config/CCDAdapterConfig.java

```diff
@@ -103,7 +108,9 @@ public ObjectMapper ccdObjectMapper() {
             .addMixIn(ClaimData.class, ClaimDataMixIn.class)
             .addMixIn(ContactDetails.class, ContactDetailsMixIn.class)
             .addMixIn(Claim.class, ClaimMixIn.class)
-            .addMixIn(Representative.class, RepresentativeMixIn.class);
+            .addMixIn(Representative.class, RepresentativeMixIn.class)
+            .addMixIn(ClaimantResponse.class, ClaimantResponseMixin.class)
+            .addMixIn(PaymentIntention.class, PaymentIntentionMixin.class);
```

Comment:
As a matter of convention, you might find it helpful to keep these in alphabetical order so it's easier to compare with file listings to see if anything's missing.